mne/epochs.py

```diff
@@ -342,6 +352,8 @@ class BaseEpochs(ProjMixin, ContainsMixin, UpdateChannelsMixin, ShiftTimeMixin,
     data : ndarray | None
         If ``None``, data will be read from the Raw object. If ndarray, must be
         of shape (n_epochs, n_channels, n_times).
+    raw_sfreq : float
+        The original Raw object sampling rate.
```

Comment:
Please add this last after a `*`, like `*, raw_sfreq=None`. If None, it should just use `info['sfreq']`. This will have better backward compatibility, e.g., prevent breaking `RtEpochs` in MNE-Realtime. This will also make it so you can revert a number of the test changes here, and maybe even some of the other BaseEpochs subclass changes

Reply:
Done.